sklearn/metrics/tests/test_pairwise.py

```diff
@@ -252,10 +255,16 @@ def callable_rbf_kernel(x, y, **kwds):
 @pytest.mark.parametrize('array_constr', [np.array, csr_matrix])
 @pytest.mark.parametrize('dtype', [np.float64, int])
 def test_pairwise_parallel(func, metric, kwds, array_constr, dtype):
+    # wminkowki metric will be deprecated starting from scipy 1.6
+    # and removed in scipy 1.8
+    if version.short_version >= '1.6.0' and (
```

Comment:
If think we should avoid comparing version strings directly because the lexicographical order does not always match the version order (when their are double digits components).

```python
from sklearn.utils.fixes import sp_version, parse_version

sp_version >= parse_version("1.6.0")
```